src/webchat-ui/components/WebchatUI.tsx

```diff
@@ -768,14 +799,16 @@ export class WebchatUI extends React.PureComponent<
 			<>
 				<Header
 					onClose={handleOnClose}
-					onGoBack={handleOnGoBack}
+					onGoBack={!showInformationMessage ? handleOnGoBack : undefined}
```

Comment:
It's better for readability to not have an inversion if possible:
```javascript
onGoBack={showInformationMessage ? undefined : handleOnGoBack}
```

Reply:
fixed